src/style.css

```diff
@@ -1 +1,32 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 22px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  height: 116px;
```

Comment:
All 7 tests fail:
![stars-failure](https://github.com/mate-academy/layout_stars/assets/6094468/474f3d93-6af6-4d04-b1b4-4fadb2870717)
Looks like that's because the distance between rows and the padding is different from expected design. I assume that incorrect padding and height values are causing this. Please take a look at that, most likely these properties can be removed at all.